src/Addons/34thPRC_Main/config.cpp

```diff
@@ -0,0 +1,25 @@
+class CfgPatches
+{
+	class main
+	{
+		name = "main";
+		author = "34th PRC Modding Team";
+		units[] = {};
+		weapons[] = {};
+		requiredVersion = "0.21.0"; //Update to match changelog version on release
```

Comment:
```suggestion
```
This is the version of Arma that our mod required; and, apparently, is no longer used by the game?